official/recommendation/neumf_model.py

```diff
@@ -199,29 +199,39 @@ def construct_model(users, items, params):
   batch_size = user_input.get_shape()[0]
 
   if params["use_tpu"]:
+    cmb_embedding_user = tf.get_variable(
```

Comment:
You probably need these under the variable_scope. Otherwise, the variable_scope doesn't do anything at all

Reply:
Good catch. Thanks.